src/app/clusters/thread_network_diagnostics_server/thread_network_diagnostics_server.cpp

```diff
@@ -16,11 +16,63 @@
  */
 
 #include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/AttributeAccessInterface.h>
 #include <app/CommandHandler.h>
+#include <app/MessageDef/AttributeDataElement.h>
 #include <app/util/af.h>
+#include <app/util/attribute-storage.h>
+#include <lib/core/CHIPEncoding.h>
+#include <lib/core/CHIPTLVTypes.h>
+#include <lib/core/Optional.h>
```

Comment:
Is Optional.h required?

Reply:
Yes, for initializing the attribute accessor.